Dart/src/com/jetbrains/lang/dart/analyzer/DartAnalysisServerService.java

```diff
@@ -1713,6 +1796,24 @@ private static boolean awaitForLatchCheckingCanceled(@NotNull final AnalysisServ
     }
   }
 
+  private void registerPostfixCompletionTemplates() {
+    ApplicationManager.getApplication().invokeLater(() -> {
+      // There must be a better way to initialize postfix templates!
+      for (int n = 20; n > 0; n--) {
+        if (DartPostfixTemplateProvider.initializeTemplates(this)) {
```

Comment:
@stevemessick, is this hack to make sure that the server is warmed up? Maybe catch the moment when initial 'pub list' or initial analysis is done (see around `serverStatus()` method)?

Also, as `DartPostfixTemplateProvider.initializeTemplates()` is called on server startup is there a need to call it again in `DartRemotePostfixTemplate.isApplicable()`?